CIScripts/Test/Utils/ComponentsInstallation.ps1

```diff
@@ -33,6 +38,12 @@ function Uninstall-Agent {
 
     Write-Host "Uninstalling Agent"
     Invoke-MsiExec -Uninstall -Session $Session -Path "C:\Artifacts\contrail-vrouter-agent.msi"
+
+    # Get rid of all leftover handles to the Agent service
+    Invoke-Command -Session $Session -ScriptBlock {
```

Comment:
Maybe this should be part of Invoke-MsiExec?

Reply:
Only agent registers a service so this seems more Agent specific. But I'm open to doing it other way.

Reply:
Actually DockerDriver is also registered as a service, moreover someday it will be run as a service, so I would suggest to move it inside Invoke-MsiExec

Reply:
(there were also plans to have node-manager service and forwarding-extension service, maybe orchestrator in the future as well. So this would future proof them)